draft-ietf-hpke-pq.md

```diff
@@ -333,6 +333,17 @@ including both implementation flaws as well as new cryptanalysis. See
 {{?I-D.irtf-cfrg-hybrid-kems}} for further analysis of hybrid security
 properties.
 
+## Asymmetric-key-authenticated modes of RFC9180
+
+In the {{?RFC9180}} version of HPKE, KEMs could optionally define the
+additional functions `AuthEncap` and `AuthDecap`. These functions allowed
+a sender to authenticate the message to the recipient without interaction.
+
+The KEMs defined in this document do not support `AuthEncap`/`AuthDecap` and
+can not be used to migrate uses of HPKE that rely on this mode.
+Pre-shared-key-authenticated HPKE ({{!I-D.ietf-hpke-hpke}}, Section 5.1.2)
+or (post-quantum) digital signatures may be suitable alternatives.
```

Comment:
```suggestion
PSK-authenticated HPKE ({{Section 5.1.2 of HPKE}})
or digital signatures may be suitable alternatives.
```

(Note: `{{HPKE}}` will break the build until #20 lands.)

Reply:
That doesn't exist, sadly.  I've considering doing that work, but it's not a simple thing to do.  So, obviously, this will rot.